app/src/main/java/org/feature/fox/coffee_counter/ui/common/NormalTextField.kt

```diff
@@ -0,0 +1,28 @@
+package org.feature.fox.coffee_counter.ui.common
+
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Text
+import androidx.compose.material.TextFieldDefaults
+import androidx.compose.runtime.*
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+import org.feature.fox.coffee_counter.ui.theme.CrayolaBrown
+
+@Composable
+fun NormalTextField(text: String = "", label: String) {
```

Comment:
The normal TextField isn't so normal anymore if we make it Outlined.
We could rename it to "CommonTextField" or something similiar.

Reply:
Should be resolved in 936b74b05378a10e3507aed9cebb27dd1778415e